src/mod/applications/mod_callcenter/mod_callcenter.c

```diff
@@ -3225,11 +3225,15 @@ SWITCH_STANDARD_APP(callcenter_function)
 		agent_found = switch_true(p);
 	}
 
-	/* Stop member thread */
+	/* Stop member thread and announcement*/
 	if (h) {
 		h->running = 0;
 	}
 
+	if (switch_channel_get_private(member_channel, queue->announce)) {
```

Comment:
At this point, the channel has no media, so displace already stopped. No reason to do this.